lang/cs.json

```diff
@@ -1,26 +1,85 @@
 {
+  "Audio Player": "Audio Přehravač",
+  "Video Player": "Video Přehravač",
   "Play": "Přehrát",
   "Pause": "Pauza",
+  "Replay": "Spustit znovu",
   "Current Time": "Aktuální čas",
   "Duration": "Doba trvání",
   "Remaining Time": "Zbývající čas",
-  "Stream Type": "Stream Type",
+  "Stream Type": "Typ streamu",
   "LIVE": "ŽIVĚ",
   "Loaded": "Načteno",
   "Progress": "Stav",
+  "Progress Bar": "Postupní pruh",
+  "progress bar timing: currentTime={1} duration={2}": "{1} z {2}",
   "Fullscreen": "Celá obrazovka",
   "Non-Fullscreen": "Zmenšená obrazovka",
   "Mute": "Ztlumit zvuk",
   "Unmute": "Přehrát zvuk",
   "Playback Rate": "Rychlost přehrávání",
   "Subtitles": "Titulky",
-  "subtitles off": "Titulky vypnuty",
+  "subtitles off": "Bez titulků",
   "Captions": "Popisky",
   "captions off": "Popisky vypnuty",
   "Chapters": "Kapitoly",
+  "Descriptions": "Popisky",
```

Comment:
Again, using Google Translate, "Descriptions for the Blind" comes back as "Popisy pro nevidomé" - does that work?

Reply:
"Popisky" for captions is perfect! With description it works for both, "popisek" is more like more like describe, so it may make more sense.